shared/xccdf/services/nfs.xml

```diff
@@ -86,17 +86,17 @@ this service should be disabled.
 
 </Group> <!-- end disabling_nfs -->
 
-<Group id="nfs_configuring_all_machines">
+<Group id="nfs_configuring_all_systems">
```

Comment:
I am not sure how strongly I feel about this but changing IDs breaks existing tailoring files for our users. We may want to compromise and leave machines in the ID. What do you think?